go.mod

```diff
@@ -1,9 +1,6 @@
 module github.com/ubuntu/authd-oidc-brokers
 
 go 1.24.0
-
-toolchain go1.24.1
-
```

Comment:
Huh, no idea why dependabot all of a sudden removes the toolchain from the `go.mod`. Have you seen that before, @denisonbarbosa?

Reply:
If I manually run `go get github.com/microsoftgraph/msgraph-sdk-go` on `main` it does not remove the toolchain from `go.mod`.

Reply:
I suspect that https://github.com/dependabot/dependabot-core/issues/11825 is somehow related.

Reply:
(I hope that will happen soon because it seems to break a log of Go projects on GitHub)